api/database/migrations/003_sar_side_car_files.up.sql

```diff
@@ -0,0 +1,4 @@
+
+ALTER TABLE media
+  ADD COLUMN IF NOT EXISTS side_car_path varchar(1024) NOT NULL DEFAULT '',
+  ADD COLUMN IF NOT EXISTS side_car_hash varchar(32) NOT NULL DEFAULT '';
```

Comment:
Maybe we should use NULL here to indicate the sidecar file doesn't exist?

Reply:
Shall do, I didn't originally because I wasn't sure how to handle having a nil or string but looks like a *string will work :)